conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -32,6 +32,11 @@ type algodImporter struct {
 	cancel  context.CancelFunc
 }
 
+func (algodImp *algodImporter) OnComplete(input data.BlockData) error {
+	_, err := algodImp.aclient.SetSyncRound(input.Round() + 1).Do(algodImp.ctx)
```

Comment:
what if input.Round() is < current round in ledger? I think you would also need to call SetSyncRound in Init? 
and does algod node need to have a large enough `MaxAcctLookback` for indexer to import older rounds?

Reply:
The node will be required to have the delta in the cache. This means the indexer's round must be within `MaxAcctLookback` of the node's round. The data node will [set the sync round](https://github.com/algorand/go-algorand/pull/5009/files#diff-c10bfda5adaaf4573b0ae153fe9eba6a3be539853363b7b630d2f6da5f94c9adR118) to the latest round on startup to make sure it doesn't get ahead of the indexer.